src/lib/Components/ArtworkGrids/RelayConnections/GeneArtworksGrid.tsx

```diff
@@ -1,4 +1,4 @@
-import { createPaginationContainer, graphql } from "react-relay"
+import { ConnectionData, createPaginationContainer, graphql } from "react-relay"
 import InfiniteScrollArtworksGrid from "../InfiniteScrollGrid"
 
 import Artwork from "../Artwork"
```

Comment:
somewhat unrelated to this PR. Is it still required to for Relay to work properly? (The unused import)

Reply:
Oh yeah, I don’t actually think so, good point, I will file a ticket for it to follow-up in 4.0.1

Reply:
https://github.com/artsy/emission/issues/946